tests/tests-memcached/src/main/java/com/hazelcast/simulator/memcached/MemcachedClientFactory.java

```diff
@@ -0,0 +1,65 @@
+package com.hazelcast.simulator.memcached;
+
+import net.spy.memcached.MemcachedClient;
+import org.apache.log4j.Logger;
+
+import java.io.IOException;
+import java.net.InetSocketAddress;
+import java.util.ArrayList;
+import java.util.List;
+
+public class MemcachedClientFactory {
+
+    protected final Logger logger = Logger.getLogger(getClass());
+
+    private List<MemcachedClient> clients = new ArrayList<MemcachedClient>();
+    private List<InetSocketAddress> addresses;
+    private boolean useSharedClient;
+
+    public MemcachedClientFactory(String[] nodes) {
+        List<InetSocketAddress> addresses = new ArrayList<InetSocketAddress>();
```

Comment:
No need to have an intermediate variable. Just used the field 'addresses' directly. Also create the list when the addresses field is defined.